Hazel/vendor/Glad/premake5.lua

```diff
@@ -1,25 +1,38 @@
 project "Glad"
-    kind "StaticLib"
-    language "C"
-    
-    targetdir ("bin/" .. outputdir .. "/%{prj.name}")
-    objdir ("bin-int/" .. outputdir .. "/%{prj.name}")
+	kind "StaticLib"
+	language "C"
+	
+	targetdir ("bin/" .. outputdir .. "/%{prj.name}")
+	objdir ("bin-int/" .. outputdir .. "/%{prj.name}")
 
-    files
-    {
-        "include/glad/glad.h",
-        "include/KHR/khrplatform.h",
-        "src/glad.c"
-    }
+	files
+	{
+		"include/glad/glad.h",
+		"include/KHR/khrplatform.h",
+		"src/glad.c"
+	}
 
 	includedirs
 	{
 		"include"
 	}
-    
-    filter "system:windows"
-        systemversion "latest"
-        staticruntime "On"
+
+	filter "action:xcode4"
+		sysincludedirs
+		{
+			"${PROJECT_DIR}/include"
+		}
+	
+	filter "system:windows"
+		systemversion "latest"
+		staticruntime "On"
+
+	filter "system:macosx"
+		systemversion "latest"
+		staticruntime "On"
         
-    filter { "system:windows", "configurations:Release" }
-        buildoptions "/MT"
\ No newline at end of file
+	filter { "system:windows", "configurations:Release" }
+		buildoptions "/MT"
+
+	filter { "system:macosx", "configurations:Release" }
+		buildoptions "/MT"
```

Comment:
Why enable MT build options flag when `staticruntime` is on?

Reply:
My bad... I've removed this now.